README.md

```diff
@@ -81,8 +81,9 @@ This project uses [mocha](http://visionmedia.github.io/mocha/) as a unit testing
 * `grunt watch` - will automatically build the project on any changes.
 * `grunt lint` - ensure the code is lint free, this is also run before an `npm test`
 * `npm link` - will symlink you local working directory to the globally installed openhim-core module. Use this so you can use the global openhim-core binary to run your current work in progress. Also, if you build any local changes the server will automatically restart.
-* `grunt test --mochaGrep=<regex>` - will only run tests with names matching the regex
-* `grunt test --ddebugTests` - enabled the node debugger while running unit tests. Add `debugger` statements and use `node debug localhost:5858` to connect to the debugger instance.
+* `npm test -- --mochaGrep=<regex>` - will only run tests with names matching the regex
+* `npm test -- --debugTests` - enabled the node debugger while running unit tests. Add `debugger` statements and use `node debug localhost:5858` to connect to the debugger instance.
+* `npm test -- --bail` - exit on first test failure.
```

Comment:
Yeah :) There are actually a bunch of useful options that we loose out of when using grunt to call mocha. It sucks to have to mod the grunt file each time. I'm wondering if we should swap grunt out for plain npm scripts to build, run and test. It seems to be trend for people to use npm script more as they are simpler.